src/bitcoin.ts

```diff
@@ -49,80 +49,76 @@ const BTC_SCRIPT_TYPE_P2WPKH_V0 = 0x04;
 // `version`:   Transaction version of the inputs. All inputs must be of the same version!
 // `isSegwit`: a boolean which determines how we serialize the data and parameterize txb
 const buildBitcoinTxRequest = function (data) {
-  try {
-    const { prevOuts, recipient, value, changePath, fee } = data;
-    if (!changePath) throw new Error('No changePath provided.');
-    if (changePath.length !== 5)
-      throw new Error('Please provide a full change path.');
-    // Serialize the request
-    const payload = Buffer.alloc(59 + 69 * prevOuts.length);
-    let off = 0;
-    // Change version byte (a.k.a. address format byte)
-    const changeFmt = getAddressFormat(changePath);
-    payload.writeUInt8(changeFmt, 0);
-    off++;
+  const { prevOuts, recipient, value, changePath, fee } = data;
```

Comment:
Note that this is just the removal of the try/catch block - nothing else was changed in this function